README.md

```diff
@@ -8,129 +10,203 @@
+
+Python tools that might be useful during development are:
+
+- [`srt-stats-plotting`](https://github.com/mbakholdina/srt-stats-plotting) - A script designed to plot graphs based on SRT `.csv` statistics.
+- [`lib-tcpdump-processing`](https://github.com/mbakholdina/lib-tcpdump-processing) - A library designed to process `.pcap(ng)` [tcpdump](https://www.tcpdump.org/) or [Wireshark](https://www.wireshark.org/) trace files and extract SRT packets of interest for further analysis.
+- [`lib-srt-utils`](https://github.com/mbakholdina/lib-srt-utils) - A Python library containing supporting code for running SRT tests based on an experiment configuration.
+
+## Contributing
+
+Anyone is welcome to contribute. If you decide to get involved, please take a moment to review the guidelines:
+
+* [SRT Developer's Guide](docs/dev/developers-guide.md)
+* [Contributing](CONTRIBUTING.md)
+* [Reporting Issues](docs/dev/making-srt-better.md)
 
-[Homebrew](https://brew.sh/) supports "srt" formula.
+For information on contributing to the [Internet Draft](https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-01) or to submit issues please go to the following [repo](https://github.com/Haivision/srt-rfc). The repo for contributing in [SRT CookBook](https://srtlab.github.io/srt-cookbook/) can be found [here](https://github.com/SRTLab/srt-cookbook/).
 
-```shell
-brew update
-brew install srt
-```
+## License
 
-If you prefer using a head commit of `master` branch, you should add `--HEAD` option
-to `brew` command.
+By contributing code to the SRT project, you agree to license your contribution under the [MPLv2.0 License](LICENSE).
 
-```shell
-brew install --HEAD srt
-```
+## Release History
 
-Also, SRT can be built with `cmake` and `make` on Mac.
-Install cmake and openssl with development files from "brew". Note that the
-system version of OpenSSL is inappropriate, although you should be able to
-use any newer version compiled from sources, if you prefer.
+- [Release notes](https://github.com/Haivision/srt/releases)
+- [SRT versioning](./docs/dev/developers-guide.md#versioning)
 
-```shell
-brew install cmake
-brew install openssl
-export OPENSSL_ROOT_DIR=$(brew --prefix openssl)
-export OPENSSL_LIB_DIR=$(brew --prefix openssl)"/lib"
-export OPENSSL_INCLUDE_DIR=$(brew --prefix openssl)"/include"
-./configure
-make
-```
 
-### Build on Windows
+[^1]: The term “live streaming” refers to MPEG-TS style continuous data transmission with latency management. Live streaming based on segmentation and transmission of files like in HTTP Live Streaming (HLS) protocol as described in RFC8216 is not part of this use case. Note that SRT is content agnostic meaning that any type of data (not necessarily limited to MPEG-TS) can be transmitted via its payload.
```

Comment:
```suggestion
[^1]: The term “live streaming” refers (but not limited) to MPEG-TS style continuous data transmission with latency management. Live streaming based on segmentation and transmission of files like in HTTP Live Streaming (HLS) protocol as described in RFC8216 is not part of this use case. Note that SRT is content agnostic meaning that any type of data (not necessarily limited to MPEG-TS) can be transmitted via its payload.
```

Reply:
I don't think it's required. It's said about "MPEG-TS style", not exclusively MPEG-TS format. Maybe this style could be explained differently, like "data sent with the same average rate as being produced and expected to be received". @stevomatthews maybe you have some better idea how to say that.

Reply:
```suggestion
[^1]: The term “live streaming” refers to continuous data transmission (MPEG-TS or equivalent) with latency management, which is to say where the average transmission rate is expected to be the same for the sender and the receiver. Live streaming based on segmentation and transmission of files like in HTTP Live Streaming (HLS) protocol as described in RFC8216 is not part of this use case. Note that SRT is content agnostic, meaning that any type of data (not necessarily limited to MPEG-TS) can be transmitted via its payload.
```